### PR TITLE
[SMALLFIX] Use static imports for test utilities

### PR DESCRIPTION
**Pull request by lengjiayi**

### CODE REVIEW

File: core/client/fs/src/test/java/alluxio/client/block/stream/GrpcDataWriterTest.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
 import static org.mockito.Mockito.atLeastOnce;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
+import static org.junit.Assert.assertEquals;
```
Comment (aaudiber) [inline core/client/fs/src/test/java/alluxio/client/block/stream/GrpcDataWriterTest.java]: @lengjiayi please alphabetize these imports

File: core/client/fs/src/test/java/alluxio/client/block/stream/GrpcDataWriterTest.java [hunk 2]
```diff
@@ -289,10 +291,11 @@ private long verifyWriteRequests(final BlockWorkerClient client, final long chec
...
-      Assert.assertEquals(offset, request.getCommand().getOffset());
+      assertEquals(BLOCK_ID, request.getCommand().getId());
+      assertEquals(offset, request.getCommand().getOffset());
     } else {
-      Assert.assertTrue(request.hasChunk());
+      assertTrue(request.hasChunk());
     }
   }
 }
+
```
Comment (aaudiber) [inline core/client/fs/src/test/java/alluxio/client/block/stream/GrpcDataWriterTest.java]: remove extra newline

Comment (lengjiayi) [reply core/client/fs/src/test/java/alluxio/client/block/stream/GrpcDataWriterTest.java]: Thanks for your advice, I have resolved these problem and hope it meets your request.

**Comment by aaudiber** [comment]
alluxio-bot, merge this please

**Pull request merged**